__tests__/servertesting.test.js

```diff
@@ -576,3 +576,29 @@ describe("/api/articels/?topic", () => {
       });
   });
 });
+describe("/api/articles/:article_id?comment_count", () => {
+  it("200: Should return article object with comment_count attached with correct value", () => {
+    return request(app)
+      .get("/api/articles/9/?comment_count")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles[0].comment_count).toBe(2);
```

Comment:
You should just have just one article coming back, return rows[0] in your model to rid yourself of the unnecessary array